cms/djangoapps/contentstore/tasks.py

```diff
@@ -58,7 +58,7 @@ def rerun_course(source_course_key_string, destination_course_key_string, user_i
             # it's possible there was an error even before the course module was created
             pass
 
-        return "exception: " + unicode(exc)
+        return ("exception: " + unicode(exc))[:999]  # don't exceed 1000 chars
```

Comment:
Hmm.. these return statements are used only by tests currently.  Is it really this line that is causing the issue?

The data that is stored in the database is actually on L51 above (see implementation of `CourseRerunState.objects.failed`).  What is the stack trace that you got with the database error?